### PR TITLE
Load 3.0 codepen examples in documentation once more

### LINKED ISSUE #2447: NONE of the codepen examples/snippets in the documentation work.
### Prerequisites

- [ ] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/);
- [ ] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [ ] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### Description

The documentation is inaccessible because none of the examples shown work. The snippets for all examples do not compile and results are blank.

### Steps to Reproduce

1. Go to https://react-jsonschema-form.readthedocs.io/en/latest/quickstart/
2. See Codepen widgets 
3. Result section of these widgets are all blank

#### Expected behavior

The examples in documentation should work and show the generated form on the right side of the Codepen widgets.

#### Actual behavior

Codepen widgets do not compile properly, right side (result) is blank or displays an error message saying "Not found".
Makes me want to pull my hair out cuz I cannot understand how to use your library.

### Version

You can usually get this information in your `package.json` or in the file URL if you're using the unpkg one.

### PR DESCRIPTION
**Pull request by wegry**
### Reasons for making this change

The React module's Component property wasn't defined in the 3.0 UMD bundle for some reason in codepen. Use ESM @rjsf/core bundle instead.

fixes #2447 

### Checklist

* [x] **I'm updating documentation**
  - [x] I've checked the rendering of the Markdown text I've added
  Screen Shot 2021-07-14 at 18 29 29
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

**Comment by epicfaace** [comment]
Thanks for fixing this. But does this mean that the UMD bundle no longer works? Is there a potential fix for that issue as well?

**Comment by wegry** [comment]
@epicfaace I don't know. I haven't used UMD consciously in years if ESM is available. I'd expect if UMD was broken, it'd get raised as an issue otherwise.

**Comment by wegry** [comment]
Alright, looks like the `external` change I had made to the webpack.config.js was assuming the UMD bundle would be used from a bundler/module aware system. It did _not_ account for React/ReactDOM being an ambient global (like the non-esm CDN approach).

File: docs/main.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ $(".language-jsx").each(function() {
     var prefill = {
         "title":"RJSF example",
         "tags":[],
-        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/16.13.1/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.13.1/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core@2.5.1/dist/react-jsonschema-form.js"],
+        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/17.0.2/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/17.0.2/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core@3.0.1/dist/react-jsonschema-form.js"],
```
Comment (epicfaace) [inline docs/main.js:15]: Let's always get the latest version

```suggestion
        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/17.0.2/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/17.0.2/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core/dist/react-jsonschema-form.js"],
```

**Review by epicfaace — APPROVED** [review]
Great, thanks for finding the fix! :)

File: packages/core/webpack.config.dist.js [hunk 2]
```diff
@@ -14,6 +15,9 @@ module.exports = {
     libraryTarget: "umd"
   },
   plugins: [
+    new MonacoWebpackPlugin({
+      languages: ['json']
+    }),
```
Comment (wegry) [inline packages/core/webpack.config.dist.js]: I don't think this plugin is usable within a library. I'm almost certain that we're just incurring a build time cost with no usable change in output.

Comment (epicfaace) [reply packages/core/webpack.config.dist.js]: Yeah, that's right. Perhaps we could split it so it's only available for the playground and docs in a separate PR though?

**Pull request merged**